apps/etl-func/src/func.ts

```diff
@@ -85,9 +87,19 @@ const main = async (config: Config) => {
     logger,
   );
 
+  const ioComCosmosClient = new CosmosClient({
+    aadCredentials: azureCredentials,
+    endpoint: config.iocomCosmos.accountUri,
+  });
+
+  const ingestionSummaryContainer = ioComCosmosClient
+    .database(config.iocomCosmos.eventsCollectorDatabaseName)
+    .container(config.iocomCosmos.messageIngestionSummaryContainerName);
+
   const ingestMessageUseCase = new IngestMessageUseCase(
     messageAdapter,
     tokenizerClient,
+    new CosmosWeeklyEventCollector(ingestionSummaryContainer, logger),
     messageEventProducer,
   );
```

Comment:
```suggestion

    const eventCollector = new CosmosWeeklyEventCollector(ingestionSummaryContainer, logger);

  const ingestMessageUseCase = new IngestMessageUseCase(
    messageAdapter,
    tokenizerClient,
    eventCollector,
    messageEventProducer,
  );
```

Why not initializing it before as all the other parameters?

Reply:
[871c047](https://github.com/pagopa/io-messages/pull/85/commits/871c047ad6c032b91ac30903231cfe2b7efbbab2)